src/stampit.js

```diff
@@ -219,35 +210,36 @@ stampit = function stampit(options) {
 
 function isStamp(obj) {
   return (
-  isFunction(obj) &&
-  isFunction(obj.methods) &&
-  // isStamp can be called for old stampit factory object. We should check old names (state and enclose) too.
-  (isFunction(obj.refs) || isFunction(obj.state)) &&
-  (isFunction(obj.init) || isFunction(obj.enclose)) &&
-  isFunction(obj.props) &&
-  isFunction(obj.static) &&
-  isObject(obj.fixed)
+    isFunction(obj) &&
+    isFunction(obj.methods) &&
+    // isStamp can be called for old stampit factory object. We should check old names (state and enclose) too.
+    (isFunction(obj.refs) || isFunction(obj.state)) &&
+    (isFunction(obj.init) || isFunction(obj.enclose)) &&
+    isFunction(obj.props) &&
+    isFunction(obj.static) &&
+    isObject(obj.fixed)
   );
 }
 
 function convertConstructor(Constructor) {
-  var stamp = stampit();
-  mixer.mixinChainFunctions(stamp.fixed.methods, Constructor.prototype);
-  stamp.fixed.refs = stamp.fixed.state = mixer.mergeChainNonFunctions(stamp.fixed.refs, Constructor.prototype);
-  addInit(stamp.fixed, function (opts) {
-    return Constructor.apply(opts.instance, opts.args);
-  });
+  const stamp = stampit();
+  stamp.fixed.refs = stamp.fixed.state = mergeChainNonFunctions(stamp.fixed.refs, Constructor.prototype);
+
+  mixinChainFunctions(stamp.fixed.methods, Constructor.prototype);
```

Comment:
You've changed refs and methods initialization order. But, frankly, you did it right. We should have had that order from the beginning. Thanks :+1: